### ISSUE TITLE
Datatables error on Manage Users page

### ISSUE THREAD

**Issue by slay123** [eventum/eventum#192]
Open Manage Users page (https://eventum.example.org/manage/users.php) and enjoy the error:
![screenshot 2016-09-09 18 40 10](https://cloud.githubusercontent.com/assets/12842560/18412400/304f161a-7796-11e6-9df0-f6d6c0456ab0.png)

At least, when clicking okay, list will be loaded etc. Probably DT is loaded twice.

**Comment by glensc** [comment]
Unable to reproduce in clean dev env. @balsdorf what you?

**Comment by glensc** [comment]
the tn/3 url corresponds to: https://datatables.net/manual/tech-notes/3

**Comment by glensc** [comment]
ok, perhaps the init twice becames that it's first initialized via css class:
[templates/manage/users_list.tpl.html:100](https://github.com/eventum/eventum/blob/v3.1.3/templates/manage/users_list.tpl.html#L100)


and then via javascript method:
[templates/manage/users_list.tpl.html:65](https://github.com/eventum/eventum/blob/v3.1.3/templates/manage/users_list.tpl.html#L65)

but i'm still unable to reproduce in my test env

**Comment by glensc** [comment]
nevermind, line 100 just defines the class line 65 activates. i don't see how it could be activated twice

**Comment by glensc** [comment]
wow, in that broken instance:
```
  $(function() { 
    console.log('init');
  });
```

prints:
```
init
init
```

how come `jQuery.ready()` is invoked twice?!

this also explains why pasting images from clipboard uploaded them twice.

**Comment by glensc** [comment]
fucking Smarty. it can fucking burn in hell. it's `{block ... append}` is total bullshit.

adding this to `templates/extra_header.tpl`
```
{block "javascripts" append}
  <script type="text/javascript">
  </script>
{/block}
```

random other `{block "javascripts" append}` filled twice to resulting html.

first it "appends" items in included templates in wrong order. now this. !@#$!@%#$% :pig:

**Comment by glensc** [comment]
created reproducer gist: https://gist.github.com/glensc/59d329718118dc4ef6bedacbe22406a8

showing both bugs:

given the `test.tpl`:
```smarty
{extends "manage.tpl"}
{block "javascripts" append}
{/block}
{include file="form.tpl"}
{include file="list.tpl"}
```

# includes in wrong order
```
$ git clone https://gist.github.com/59d329718118dc4ef6bedacbe22406a8.git -b test-2 test-2
$ cd test-2
$ composer install
$ php test.php 
    list.tpl
    form.tpl
$
```

# includes in wrong order and output twice
```
$ git clone https://gist.github.com/59d329718118dc4ef6bedacbe22406a8.git -b test-3 test-3
$ cd test-3
$ composer install
$ php test.php 

    list.tpl
    form.tpl

    list.tpl
    form.tpl
$
```

**Comment by glensc** [comment]
https://github.com/smarty-php/smarty/issues/303 let's see what smartY they come up with

**Comment by glensc** [comment]
smarty inability to extend templates resolved with different solution: #264
